### PR TITLE
Audit windows migration article

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #1532

- Fix link at the top

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-586f0351ea/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ac42ea54ac/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0fbc0b43e6/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-f37b58eb4d/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. My suggested edits are in the attached Word doc (and PDF copy of it). Please let me know if you have any questions. Thanks!

[Prepare to migrate a Windows server_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/595451/Prepare.to.migrate.a.Windows.server_editsKH.pdf)
[Prepare to migrate a Windows server_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/595450/Prepare.to.migrate.a.Windows.server_editsKH.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7fecec546b/how-to/)

File: content/cloud-servers/prepare-to-migrate-a-windows-server.md [hunk 1]
```diff
@@ -69,18 +65,13 @@ version.
...
+imaging or migrating a server and that any pending reboots are performed.
 
-To ensure your server comes online after the migration, confirm that
-there are no pending Windows Updates that require a reboot. Pending
-Windows updates can cause errors when run on a newly migrated server, so
-it is especially important to make sure updates are completed before
-imaging or migrating a server.
-
-**For Windows Server 2008 and 2008 R2**
+** Windows Server 2008 and 2008 R2**
```
Comment (stephamon) [inline content/cloud-servers/prepare-to-migrate-a-windows-server.md]: There's a space here between the asterisks and "Windows". Not sure if it'll impact the published formatting, but should probably be removed.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-5e5e4b301c/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**